client/js/scoreboard.js

```diff
@@ -32,9 +32,12 @@ export default class Scoreboard {
 		return li;
 	}
 
-	updateScores(players) {
+	updateScores(players, board) {
```

Comment:
There is no reason for the scoreboard to receieve the entire board as argument. Especially when the only thing you need/care about is `board.current`. Prefer a `clientLocalSnake` argument that would be a snake